doma-spring-boot-samples/doma-spring-boot-sample-testcontainers/src/test/java/org/seasar/doma/boot/sample/TestContainersConfig.java

```diff
@@ -0,0 +1,20 @@
+package org.seasar.doma.boot.sample;
+
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.boot.testcontainers.service.connection.ServiceConnection;
+import org.springframework.context.annotation.Bean;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.utility.DockerImageName;
+
+@TestConfiguration(proxyBeanMethods = false)
+public class TestContainersConfig {
```

Comment:
Create a project with Spring Initializr.
And use `TestcontainersConfiguration` class included in the generated project.